src/components/Home.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FiSearch } from 'react-icons/fi';
+import { SiWorldhealthorganization } from 'react-icons/si';
+import { getCountryData } from '../redux/country/countrySlice';
+import Header from './Header';
+
+function Home() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCountryData());
+  }, []);
+
+  const countries = useSelector((store) => store.country);
+  const [querry, setQuerry] = useState('');
+  const dataSearch = (str) => setQuerry(str);
+
+  return (
+    <section>
+      <Header />
+      <div className="search__container">
+        <span>
+          <FiSearch />
+        </span>
+        <input
+          className="country__search"
+          placeholder="Eg: Afghanistan , South Africa"
+          onChange={(e) => dataSearch(e.target.value)}
+        />
+      </div>
+      <div>
+        <h1>STATS BY COUNTRY</h1>
+      </div>
+      <div>
+        <ul className="container">
+          {countries.covidData
+            .filter((value) => value.country.toLowerCase().includes(querry.toLocaleLowerCase()))
+            .map((data) => (
+              <NavLink
+                to={`/details/${data.country}`}
+                key={data.country}
+                state={
+                  data.country === null
+                    ? ''
+                    : { country: data.country, flag: data.flag }
+                }
+              >
+                <div className="country__info">
+                  {' '}
```

Comment:
- [ ] Good job displaying the contents side by side as requested by the previous reviewer. However your desgn has an overflow and the contents are not properly aligned, kindly fix this.

Your design
![image](https://user-images.githubusercontent.com/81932478/225608209-44eea8f6-79c0-4cf0-999a-d3231d33e6e5.png)


Given design
![image](https://user-images.githubusercontent.com/81932478/225608417-fe8ddd7f-f840-40cc-9c31-fbb7a9baa111.png)

### Note that this is a mobile app and as such the mobile view should match the given design